modules/EvseManager/EvseManager.cpp

```diff
@@ -159,8 +159,8 @@ void EvseManager::ready() {
         bsp->set_ev_simplified_mode_evse_limit(true);
     }
 
-    error_handling =
-        std::unique_ptr<ErrorHandling>(new ErrorHandling(r_bsp, r_hlc, r_connector_lock, r_ac_rcd, p_evse, r_imd));
+    error_handling = std::unique_ptr<ErrorHandling>(
```

Comment:
did you consider using `std::make_unique`?

Reply:
replaced with make_unique